src/utils/WrapOpenMP.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file WrapOpenMP.h
+ * @author F. Gratl
+ * @date 4/20/18
+ */
+
+#pragma once
+
+/**
+ * Provide non-OpenMP versions of the most common OpenMP function calls,
+ * so that they don't have to be wrapped in #ifdef-s every time.
+ *
+ * Proper wrapper and renaming necessary, because of -fopenmp-simd handling of
+ * gcc.
+ *
+ * Extend when necessary.
+ */
+
+#include <cassert>
+#if defined(_OPENMP)
+#include <omp.h>
+
+inline int autopas_get_thread_num() { return omp_get_thread_num(); }
+inline int autopas_get_num_threads() { return omp_get_num_threads(); }
+inline int autopas_get_max_threads() { return omp_get_max_threads(); }
+typedef omp_lock_t autopas_lock_t;
```

Comment:
I'm using c++11's std::thread's locks for the ExceptionHandler and it is also used for the logging library. should not pose any problems, right?

Reply:
don't think so. But I do not really want want to mix omp threads with std::thread locks.